test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SchemaCompare/SchemaCompareServiceTests.cs

```diff
@@ -1865,6 +1865,8 @@ private void ValidateResultEndpointInfo(SchemaCompareEndpoint originalEndpoint,
             {
                 SchemaCompareProjectEndpoint projectEndpoint = originalEndpoint as SchemaCompareProjectEndpoint;
                 Assert.AreEqual(projectEndpoint.ProjectFilePath, resultEndpoint.ProjectFilePath);
+                Assert.AreEqual(resultEndpoint.ExtractTarget, DacExtractTarget.ObjectType);
```

Comment:
also add strings to describe these asserts so it's easier to know what failed if the tests fail